File: app/src/main/java/feelings/guide/answer/AnswerContract.java
package feelings.guide.answer;

import android.provider.BaseColumns;

public class AnswerContract implements BaseColumns {
    static final String ANSWER_TABLE = "answer";
    static final String COLUMN_QUESTION_ID = "question_id";
    static final String COLUMN_DATE_TIME = "date_time";
    static final String COLUMN_ANSWER = "answer";

    static final String[] ALL_COLUMNS = {_ID, COLUMN_QUESTION_ID, COLUMN_DATE_TIME, COLUMN_ANSWER};

    public static final String SQL_CREATE_ANSWER_TABLE = "CREATE TABLE " + ANSWER_TABLE + " (" +
            _ID + " INTEGER PRIMARY KEY," +
            COLUMN_QUESTION_ID + " INTEGER," +
            COLUMN_DATE_TIME + " TEXT," +
            COLUMN_ANSWER + " TEXT)";

    private AnswerContract() {
    }

}

File: app/src/main/kotlin/feelings/guide/answer/AnswerContract.kt
package feelings.guide.answer

import android.provider.BaseColumns

internal const val ANSWER_TABLE = "answer"
internal const val COLUMN_QUESTION_ID = "question_id"
internal const val COLUMN_DATE_TIME = "date_time"
internal const val COLUMN_ANSWER = "answer"

internal val ALL_COLUMNS = arrayOf(
        BaseColumns._ID,
        COLUMN_QUESTION_ID,
        COLUMN_DATE_TIME,
        COLUMN_ANSWER
)

const val SQL_CREATE_ANSWER_TABLE = """CREATE TABLE $ANSWER_TABLE (
    ${BaseColumns._ID} INTEGER PRIMARY KEY,
    $COLUMN_QUESTION_ID INTEGER,
    $COLUMN_DATE_TIME TEXT,
    $COLUMN_ANSWER TEXT
)"""
